modules/lib/lib-node/src/main/resources/lib/xp/node.ts

```diff
@@ -820,10 +820,6 @@ export interface Node {
     _permissions?: AccessControlEntry[];
 }
 
-/**
```

Comment:
why is it removed?

Reply:
The interface do not need to be documented, classes do, because we fetch the docs from `js` files where all types will be removed.